hw/ip/sysrst_ctrl/sysrst_ctrl.core

```diff
@@ -27,17 +27,23 @@ filesets:
       - rtl/sysrst_ctrl.sv
     file_type: systemVerilogSource
 
-  files_ascentlint_waiver:
+  files_verilator_waiver:
     depend:
       # common waivers
       - lowrisc:lint:common
-    file_type: waiver
+      - lowrisc:lint:comportable
```

Comment:
this core links in shared lint waivers.

Reply:
adc_ctrl and sysrst_ctrl changes look good. Thank you for cleaning it up.

-Eric